src/map/message.cpp

```diff
@@ -656,6 +658,95 @@ namespace message
+                                regionControl.prev    = ref<uint8>(data, start + 1);
+
+                                regionControls[i] = regionControl;
+                            }
+
+                            conquest::HandleWeeklyTallyEnd(regionControls);
+                            return;
+                        }
+
+                        if (subtype == CONQUEST_WORLD2MAP_INFLUENCE_POINTS)
+                        {
+                            const std::size_t        headerLength      = 2 * sizeof(uint8);
+                            bool                     shouldUpdateZones = ref<bool>(data, 2);
+                            uint32                   size              = ref<uint32>(data, 3);
+                            std::vector<influence_t> influencePoints   = std::vector<influence_t>(size);
+                            for (int i = 0; i < size; i++)
+                            {
+                                const std::size_t start = headerLength + sizeof(bool) + sizeof(size_t) + i * sizeof(influence_t);
+
+                                influence_t influence;
+                                influence.sandoria_influence = ref<uint16>(data, start);
+                                influence.bastok_influence   = ref<uint16>(data, start + 2);
+                                influence.windurst_influence = ref<uint16>(data, start + 4);
+                                influence.beastmen_influence = ref<uint16>(data, start + 6);
+
+                                influencePoints[i] = influence;
+                            }
+
+                            conquest::HandleInfluenceUpdate(influencePoints, shouldUpdateZones);
+                            return;
+                        }
+
+                        if (subtype == CONQUEST_WORLD2MAP_REGION_CONTROL)
+                        {
+                            const std::size_t             headerLength = 2 * sizeof(uint8);
+                            uint32                        size         = ref<uint32>(data, 2);
+                            std::vector<region_control_t> regionControls;
+                            for (int i = 0; i < size; i++)
+                            {
+                                const std::size_t start = headerLength + sizeof(size_t) + i * sizeof(region_control_t);
+
+                                region_control_t regionControl;
+                                regionControl.current = ref<uint8_t>(data, start);
+                                regionControl.prev    = ref<uint8_t>(data, start + 1);
+                                regionControls.push_back(regionControl);
+                            }
+
+                            conquest::conquestData->updateRegionControls(regionControls);
+                            return;
+                        }
+
+                        break;
+                    }
+                    default:
+                        ShowWarning("Message: unhandled regional event type %d", eventType);
```

Comment:
Done. Curious, is this a codestyle thing (makes sense) or is the functionality different without it?

Reply:
It functions because it just runs from the case statement downwards, but that doesn't align with what if and while do, so its cleaner and clearer to wrap things in {}'s, with the break either inside or outside - I don't know if we have a convention for that